tests/load/pipeline/test_model_item_format.py

```diff
@@ -43,17 +45,23 @@
     ids=lambda x: x.name,
 )
 def test_simple_incremental(destination_config: DestinationTestConfiguration) -> None:
-    pipeline = destination_config.setup_pipeline("test_model_item_format", dev_mode=False)
+    pipeline = destination_config.setup_pipeline(
+        f"test_model_item_format_{uniq_id()}", dev_mode=False
+    )
 
-    pipeline.run([{"a": i, "b": i + 1} for i in range(10)], table_name="example_table")
+    pipeline.run(
+        [{"a": i, "b": i + 1} for i in range(10)],
+        table_name="example_table",
+        table_format="iceberg",
+    )
     dataset = pipeline.dataset()
 
     select_dialect = pipeline.destination.capabilities().sqlglot_dialect
 
     example_table_columns = dataset.schema.tables["example_table"]["columns"]
 
     # TODO: incremental is not supported for models yet
-    @dlt.resource()
+    @dlt.resource(table_format="iceberg")
```

Comment:
I'd only set this for athena, or alternatively just run the athena version with iceberg enabled for all tables, you can filter out the other one. It just should not be set for all other destination tests.